baselines/sharepointonedrive.md

```diff
@@ -0,0 +1,338 @@
+
+### Policies
+#### MS.SHAREPOINT-ONEDRIVE.1.1v1
+External sharing for SharePoint SHALL be limited to Existing Guests or Only People in your Organization.
+- _Rationale:_ Sharing information outside the organization increases the risk of unauthorized disclosure. By limiting external sharing, administrators decrease the risk of unauthorized disclosure.
+- _Last modified:_ June 2023
+
+#### MS.SHAREPOINT-ONEDRIVE.1.2v1
+External sharing for OneDrive SHALL be limited to Existing Guests or Only People in your Organization.
+- _Rationale:_ Sharing files outside the organization increases the risk of unauthorized disclosure. By limiting external sharing, administrators decrease the risk of unauthorized disclosure.
+- _Last modified:_ June 2023
+
+#### MS.SHAREPOINT-ONEDRIVE.1.3v1
+External sharing SHALL be restricted to approved external domains per interagency collaboration needs.
+- _Rationale:_ By limiting sharing to domains used for interagency collaboration purposes, administrators prevent sharing with unknown organizations and individuals.
+- _Last modified:_ June 2023
+- _Note:_ This policy is only applicable if the external sharing slider on the admin page is set to any value other than Only People in your Organization.
+
+#### MS.SHAREPOINT-ONEDRIVE.1.4v1
+Only users in approved security groups SHALL be allowed to share externally.
+- _Rationale:_ By limiting sharing to approved security groups based on interagency collaboration needs, administrators ensure only approved users can use share information outside the organization.
+- _Last modified:_ June 2023
+- _Note:_ This policy is only applicable if the external sharing slider on the admin page is set to any value other than Only People in your Organization.
+
+#### MS.SHAREPOINT-ONEDRIVE.1.5v1
+Guest access SHALL be limited to the email the invitation was sent to.
+- _Rationale:_ Email invitations allow external guests to access shared information. By requiring guests to sign in using the same account where the invite was sent, administrators ensure only the intended guest can use the invite.
+- _Last modified:_ June 2023
+- _Note:_ This policy is only applicable if the external sharing slider on the admin page is set to any value other than Only People in your Organization.
+
+### Resources
+
+- [Overview of external sharing in SharePoint and OneDrive in Microsoft 365 \| Microsoft Documents](https://learn.microsoft.com/en-us/sharepoint/external-sharing-overview)
+
+- [Manage sharing settings for SharePoint and OneDrive in Microsoft 365 \| Microsoft Documents](https://docs.microsoft.com/en-us/sharepoint/turn-external-sharing-on-or-off)
+
+### License Requirements
+
+- N/A
+
+### Implementation
+
+All of the settings in this section are configured in the **SharePoint admin center**.
+
+#### MS.SHAREPOINT-ONEDRIVE.1.1v1 instructions:
```

Comment:
`#### MS.SHAREPOINT-ONEDRIVE.1.1v1 instructions:`

I might have missed the conversations about the ID for these policies, `SHAREPOINT-ONEDRIVE` works, but it is longer than I personally would like.

What if we named these policies like this?
`#### MS.SHAREPOINT.1.1v1 instructions:`

As all of OneDrive's settings are configured inside the SharePoint admin center, this seems like an acceptable abstraction and would significantly simplify the ID. Thoughts?

Reply:
Yes it is long. @Dylan-MITRE and I will put it up for a team vote at tomorrow's stand-up meeting and we can finalize it. We also can vote on what we want the product name argument to be when executing ScubaGear at the command line.

Reply:
I updated the identifiers to SHAREPOINT based on how the team voted.

Reply:
I'm resolving since Alden is on vacation.